backend/shared/cosmo_db.py

```diff
@@ -76,6 +77,35 @@ def get_report(report_id):
         logging.error(f"Unexpected error retrieving report with id '{report_id}'")
         raise
 
+#get all reports type "curation" and "companySummarization"
+def get_all_reports_curation():
```

Comment:
@ramirezmorac2  The function `get_all_reports_curation` should return either all the reports or the reports filtered by type. To better reflect its functionality, I suggest renaming the function to `get_reports_by_type` or `get_filtered_reports`,